src/sentry/conf/server.py

```diff
@@ -1336,6 +1336,14 @@ def create_partitioned_queues(name):
 # Rate limiting backend
 SENTRY_RATELIMITER = "sentry.ratelimits.base.RateLimiter"
 SENTRY_RATELIMITER_OPTIONS = {}
+SENTRY_RATELIMITER_DEFAULT_IP = 1000
+SENTRY_RATELIMITER_DEFAULT_USER = 3230
+SENTRY_RATELIMITER_DEFAULT_ORG = 620
```

Comment:
Where are these numbers from?

Reply:
Max values for each category from a week of api access data

Reply:
I can add a comment saying that the data came from analysis of a week of data

Reply:
Is this on a max per endpoint level? Seems really high for any org to be calling endpoints this much (other than /store)

Reply:
Wow, some users are hitting one of our endpoints at 50rps? That's wild...

Reply:
Where did you get the 50 rps?

Reply:
Discussed on slack, this is based on max requests in a particular second bucket rather than a consistent rps